### PR TITLE
Use inquirer instead of prompter

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: definitions/prompt.d.ts [hunk 1]
```diff
@@ -1,41 +1,25 @@
...
-}
-
-interface IPromptHistoryValue {
-	value: any;
+	type: string;
+	name: string;
+	message: any;
+	default?: any;
+	choices?: any[];
+	//validate?: (userInput: any) => any;
```
Comment (teobugslayer) [inline definitions/prompt.d.ts]: Why is this commented?

File: prompter.ts [hunk 2]
```diff
@@ -33,27 +33,22 @@ export class Prompter implements IPrompter {
...
 		var future = new Future;
-		prompt.get(schema, (err: any, result: any) => {
-			if (err) {
+		prompt.prompt(schema, (result: any) => {
+			if(result) {
+				future.return(result);
+			} else {
 				// the error is normally Ctrl-C:
 				// move the cursor after the prompt (so that the error is not printed right after the user input)
 				console.log("");
```
Comment (teobugslayer) [inline prompter.ts]: Do we need this console log yet?

**Comment by teobugslayer** [comment]
:+1:

**Comment by jasssonpet** [comment]
I think, that the package.json should be updated too.

**Pull request merged**